scripts/index.js

```diff
@@ -0,0 +1,128 @@
+const state = {
+    tempCount: 70
+};
+
+
+const updateTemp = tempValue => {
+    const tempCountContainer = document.getElementById("tempValue");
+    tempCountContainer.textContent = tempValue;
```

Comment:
I think this will fix the bug: 
```suggestion
    tempCountContainer.textContent = state.tempCount;
```